configure.ac

```diff
@@ -1829,7 +1829,7 @@ if test "$Py_LTO" = 'true' ; then
   case $CC in
     *clang*)
       dnl flag to disable lto during linking
-      LDFLAGS_NOLTO="-fno-lto"
+      LDFLAGS_NOLTO="-flto=${Py_LTO_POLICY}"
```

Comment:
`LDFLAGS_NOLTO` should be linker flags without LTO, hence the name `no lto`.

Reply:
I guess I'll need to make a bigger change, that changes the name to something more descriptive.

These are the lto flags used for building the frozen stuff and bootstrap.

For clang that seems to work better when they are the same as for the rest of the build.

Reply:
clang does not support linking with objects that have LTO information? It works fine for GCC. Could you please update the comment on line 1831?

Reply:
I am not quite sure what the exact rootcause is.  I just noticed that when I tried to build some things with LTO and some things (like `_bootstrap_python`) without LTO, I get an error from the linker via clang.

Reply:
I updated the comment.

Reply:
Does thin LTO linking work with object files that are built with full LTO? This would speed up LTO builds a bit.

Reply:
IIUC, at this moment ``-flto=${Py_LTO_POLICY}`` could be ``-flto=default`` so it's wrong.

Reply:
Hmm, let me think about how to fix this.  Or do you have a suggestion?